### ISSUE TITLE
[BUG] to logical is broken for very aggregated

### ISSUE THREAD

**Issue by DenisYaroshevskiy** [jfalcou/eve#1666]
This: https://github.com/jfalcou/eve/blob/c8686c45ad32d527eff3743bd5cbb7bfede0e1bd/include/eve/detail/function/simd/common/to_logical.hpp#L67

Will run into problems very quickly

**Comment by jfalcou** [comment]
Philosphical question : do we want to support VERY aggregated types after all ?

**Comment by DenisYaroshevskiy** [comment]
Yeah. I don't know an answer to that. I'm generally not sure about aggregation, since it's not very good. Or like 64 bytes max.

Needs a decision.

**Comment by jfalcou** [comment]
wide larger than cache lien size is useless IMHO. SO we could have : 
 - a macimum size in a static_assert
 - a hand unrolled implementation of aggregation faster to compile
